test/proposals/mips/mip-t01/mip-t01.sol

```diff
@@ -0,0 +1,96 @@
+//SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.19;
+
+import "@forge-std/Test.sol";
+
+import {MToken} from "@protocol/MToken.sol";
+import {Configs} from "@test/proposals/Configs.sol";
+import {ChainIds} from "@test/utils/ChainIds.sol";
+import {Proposal} from "@test/proposals/proposalTypes/Proposal.sol";
+import {Addresses} from "@test/proposals/Addresses.sol";
+import {TimelockProposal} from "@test/proposals/proposalTypes/TimelockProposal.sol";
+import {CrossChainProposal} from "@test/proposals/proposalTypes/CrossChainProposal.sol";
+import {ChainlinkOracle} from "@protocol/Oracles/ChainlinkOracle.sol";
+
+/// This MIP sets the price feeds for wstETH and cbETH.
+contract mipt01 is Proposal, CrossChainProposal, ChainIds, Configs {
+    string public constant name = "MIPT01";
+
+    constructor() {
+        _setNonce(2);
+    }
+
+    function deploy(Addresses addresses, address) public override {}
```

Comment:
the deploy function should handle deployment of `stETHETH_ORACLE ` and `cbETHETH_ORACLE`

Reply:
they're already deployed. I don't want to deploy new ones, I just want to set the price feed.